cloudmarker/alerts/slackalert.py

```diff
@@ -0,0 +1,108 @@
+"""Alerter to send Slack messages for identified anomalies."""
+
+import json
+import logging
+
+import slackclient
+
+_log = logging.getLogger(__name__)
+
+
+class SlackAlert:
+    """Alert plugin to send Slack notifications."""
+
+    def __init__(self, bot_user_token, to, text,
+                 temp_file='/tmp/cloudmarker/slackanomalyreport.json'):
```

Comment:
Please rename the file path to `/tmp/cloudmarker/slackalert.json` to keep it simple and consistent with the plugin name.